crypto/x509/v3_genn.c

```diff
@@ -58,6 +58,35 @@ GENERAL_NAME *GENERAL_NAME_dup(const GENERAL_NAME *a)
                                     (char *)a);
 }
 
+int GENERAL_NAME_set1_X509_NAME(GENERAL_NAME **tgt, const X509_NAME *src)
+{
+    GENERAL_NAME *name;
```

Comment:
```suggestion
    GENERAL_NAME *name = NULL;
```

Reply:
Unnecessary as this is assigned a few lines below.